ActivityPub.Types/AS/Collection/ASCollection.cs

```diff
@@ -98,6 +99,32 @@ public int TotalItems
     public bool HasItems => Items?.Any() == true;
 
     public static implicit operator ASCollection(List<ASObject> collection) => new() { Items = new LinkableList<ASObject>(collection) };
+
+    public IEnumerator<Linkable<ASObject>> GetEnumerator()
+    {
+        if (Items == null)
+        {
+            yield break;
+        }
+
+        foreach (var item in Items)
+        {
+            if (item.HasValue)
```

Comment:
If the previous change is not possible, then can we yield `item` directly? This version will churn memory by re-wrapping the link/value into a new Linkable.